File: nodejs/handler.js
'use strict';
var AWS = require("aws-sdk");
AWS.config.update({ region: 'us-east-2' });

module.exports.hello = async (event) => {
  var body = JSON.parse(event.body);
  var docClient = new AWS.DynamoDB.DocumentClient();
  var params = {
    TableName: "Assets",
    Key: {
      "ContractId": body.ContractId
    }
  }
  var data = await docClient.get(params).promise();
  
  return {
    statusCode: 200,
    body: JSON.stringify(data.Item, null, 2)
  };
};

File: python3/handler.py
from __future__ import print_function
import json
import boto3
import decimal

# This is because for some reason, json dumps does not support decimal 
class DecimalEncoder(json.JSONEncoder):
    def default(self, o):
        if isinstance(o, decimal.Decimal):
            if o % 1 > 0:
                return float(o)
            else:
                return int(o)
        return super(DecimalEncoder, self).default(o)

def hello(event, context):
    dynamodb = boto3.resource('dynamodb')
    table = dynamodb.Table('Assets')
    body = json.loads(event['body'])

    res = table.get_item(
        Key={
            'ContractId': body['ContractId']
        }
    )

    item = res['Item']

    response = {
        "statusCode": 200,
        "body": json.dumps(item, cls=DecimalEncoder)
    }

    return response
